docs/dev-cloudtrail-initial-setup.rst

```diff
@@ -0,0 +1,112 @@
+***********************************
+AWS S3 and SQS setup for CloudTrail
+***********************************
+
+S3 setup
+========
+#. log in to the AWS web console for your cluster account
+#. go to the s3 (Simple Storage Service) page
+#. click "Create bucket"
+#. enter a unique name for your bucket ("yourname-cloudigrade-s3")
+#. select a region (us-east-1) and click "Next"
+#. the default properties and permissions are OK, click "Next" until you reach the "Review" step
+#. click "Create bucket"
+
+SQS setup
+=========
+#. navigate to the SQS (Simple Queue Service) page
+#. click "Create New Queue"
+#. give the queue a reasonably unique name ("yourname-cloudigrade-sqs")
+#. select Standard Queue and click "Quick-Create Queue"
+#. select your queue and click on the "Permissions" tab
+#. select "Edit Policy Document (Advanced)"
+#. click JSON to switch to the text editor.
+#. replace the contents of the text editor with the following:
+
+   .. code-block:: json
+
+       {
+          "Version": "2012-10-17",
+          "Id": "unique-identifier",
+          "Statement": [
+            {
+              "Sid": "unique-identifier",
+              "Effect": "Allow",
+              "Principal": {
+                "AWS": "*"
+              },
+              "Action": "SQS:SendMessage",
+              "Resource": "arn-for-your-queue",
+              "Condition": {
+                "ArnLike": {
+                  "aws:SourceArn": "arn-for-your-bucket"
+                }
+              }
+            }
+          ]
+        }
+
+   .. note::
+       change the "Id" and "Sid" values to reflect unique identifiers. You must also replace the resource arns with the arns to your queue and to your s3 bucket respectively
+#. click "Review Policy"
+#. click "Save Changes"
+
+s3 configuration
+================
+#. navigate back to the s3 page
+#. click on your s3 bucket
+#. select the "Permissions" tab
+#. select "Bucket Policy"
+#. replace the contents of the text editor with the following:
+
+   .. code-block:: json
+
+       {
+            "Version": "2012-10-17",
+            "Statement": [
+                {
+                    "Sid": "unique-id",
+                    "Effect": "Allow",
+                    "Principal": {
+                        "Service": "cloudtrail.amazonaws.com"
+                    },
+                    "Action": "s3:GetBucketAcl",
+                    "Resource": "arn-for-your-bucket"
```

Comment:
Make `arn-for-your-bucket` a variable in the template so its obvious you need to change it.  For example, instead of `arn-for-your-bucket` use `CHANGE_TO_ARN_FOR_YOUR_BUCKET`